src/main/java/de/metas/ui/web/pporder/process/WEBUI_PP_Order_HUEditor_Launcher.java

```diff
@@ -78,20 +78,22 @@ protected ProcessPreconditionsResolution checkPreconditionsApplicable()
 		}
 
 		final PPOrderLinesView ppOrderLineView = getView();
-		if (!(ppOrderLineView.isStatusPlanning() || ppOrderLineView.isStatusReview()))
+		final PPOrderLineRow singleSelectedRow = getSingleSelectedRow();
+
+		final boolean isIssueOnlyWhatWasReveiced = X_PP_Order_BOMLine.ISSUEMETHOD_IssueOnlyForReceived.equals(singleSelectedRow.getIssueMethod());
+
+		if (!(ppOrderLineView.isStatusPlanning() || ppOrderLineView.isStatusReview()) && !isIssueOnlyWhatWasReveiced)
```

Comment:
this expression is overly complex. afaiu it might as well be 
```java
!ppOrderLineView.isStatusPlanning() && !ppOrderLineView.isStatusReview() && !isIssueOnlyWhatWasReveiced
```
or 
```java
!(ppOrderLineView.isStatusPlanning() || ppOrderLineView.isStatusReview() || isIssueOnlyWhatWasReveiced)
```

also, the reject resaon now doesn't reflect the expression anymore

Reply:
Done